index.html

```diff
@@ -181,6 +181,23 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+    var fireballSize = 22;
+    var getFireballSpeed = function (left) {
+        left ? 5 : 2;
```

Comment:
You should test your program before you make a pull request. The game doesn't work for now. Figure out why (actually I pointed the problem already by this comment), make corrections, commit them and push it to origin. Pull request will refresh automatically after your push.